File: programmers/pro12906.py
# 같은 숫자는 싫어

def solution(arr):
    answer = []
    answer.append(arr[0])
    for value in arr:
        if(answer[len(answer)-1] != value):
            answer.append(value)
   
    # [실행] 버튼을 누르면 출력 값을 볼 수 있습니다.
    
    return answer

File: javascript/pro12906.js
// 쉬운거부터 익숙해지기! 
// 프로그래머스 level1 같은 숫자는 싫어

function solution(arr) {
    let answer = [];
    let lastValue;
    arr.map((value) => {
        if (lastValue !== value){
            answer.push(value)
        }
        lastValue = value
    })
    
    return answer;
}